build.rs

```diff
@@ -154,6 +154,10 @@ fn write_script<W: io::Write>(w: &mut W) -> Result<()> {
 #
 
 set -e
+
+git stash --keep-index --include-untracked --quiet
```

Comment:
I have two points here.

- I agree with #3, however, `--include-untracked` would take time when there is a heavy directory such as `node_modules`. What is the case untracked file should be stashed?
- I think showing what hook script does is preferred rather than doing it implicitly. It is important when these lines don't work as we intended. We can analyze what did not work. I prefer to adding `echo` to show what is being done.

```sh
echo '+git stash --keep-index --include-untracked'
git stash --keep-index --include-untracked
```

Reply:
Will do, should the trap echo too?

Reply:
Yeah, I feel so.

Reply:
Everything should echo now. Also removed the `--quiet`